gulpfile.js

```diff
@@ -1154,7 +1154,7 @@ gulp.task("types", function (done) {
     "forceConsistentCasingInFileNames",
     "emitDeclarationOnly",
   ].join(" --");
-  exec(`node_modules/.bin/tsc --${args} src/pdf.js`, done);
+  exec(`"node_modules/.bin/tsc" --${args} src/pdf.js`, done);
```

Comment:
If this fixes things on Windows, shouldn't https://github.com/mozilla/pdf.js/blob/71f8e1f538386fcbdb31d8cc098572ce32bf5c56/gulpfile.js#L1377 be changed too?

Reply:
Whoops, good catch :-)